app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/HorizontalSnapScrollView.java

```diff
@@ -27,6 +28,8 @@
 
     private int columnWidth;
 
+    private int roomsCount = Integer.MIN_VALUE;
```

Comment:
I'd use a custom constant for this. Something like `NOT_INITIALIZED`.

Reply:
Good idea. Will do.

Reply:
:heavy_check_mark: Changed in d54ccfe4ba88eef66038bc2a271e8bc8a1460732.